packages/editor-ui/src/plugins/i18n/locales/en.json

```diff
@@ -2295,6 +2297,8 @@
 	"workflows.empty.browseTemplates": "Explore workflow templates",
 	"workflows.empty.learnN8n": "Learn n8n",
 	"workflows.empty.button.disabled.tooltip": "Your current role in the project does not allow you to create workflows",
+	"workflows.empty.easyAI": "Test a ready-to-go AI Agent example",
```

Comment:
The line break is unfortunate between AI and Agent, can't figure out a nice suggestion though 😓

Reply:
Added a bit if horizontal padding to the label to make it break earlier:
![image](https://github.com/user-attachments/assets/ff388cd4-3738-40c6-b493-8e28302da9d7)